xpi-manifest.yml

```diff
@@ -185,9 +185,9 @@ xpis:
       - web-ext-artifacts/news-disinformation-study.xpi
     addon-type: privileged
     install-type: npm
-  - name: ion-core-addon
-    description: Ion Core Add-on
-    repo-prefix: ioncoreaddon
+  - name: rally-core-addon
+    description: Rally Core Add-on
+    repo-prefix: rallycoreaddon
     active: true
     private-repo: false
     artifacts:
```

Comment:
We aren't renaming the file? If we do, the package.json needs to change as well. https://github.com/mozilla-extensions/rally-core-addon/blob/master/package.json

Reply:
> We aren't renaming the file? If we do, the package.json needs to change as well. https://github.com/mozilla-extensions/rally-core-addon/blob/master/package.json

Yes I think that's true, we need to change a few other things in https://github.com/mozilla-rally/core-addon/pull/271 (like the add-on ID potentially as well)

Reply:
There is a PR that I think addresses this (are you referring to the name in the `package.json`?): https://github.com/mozilla-rally/core-addon/pull/271

Reply:
Whoops, sorry @escapewindow . This took less than expected, so now we have #95 up for review. Thank you for your patience and sorry for the noise.